File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/PageVo.java
package org.javamaster.invocationlab.admin.model.erd;

import com.alibaba.fastjson.JSONObject;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 * @date 2023/2/14
 */
@NoArgsConstructor
@Data
@AllArgsConstructor
@Builder
public class PageVo {

    private List<RecordsVo> records;
    private Integer total;
    private Integer size;
    private Integer current;
    private List<JSONObject> orders;
    private Boolean searchCount;
    private Integer pages;

}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/PageVo.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

import com.alibaba.fastjson.JSONObject

/**
 * @author yudong
 * @date 2023/2/14
 */
@AllOpen
class PageVo {
    var records: MutableList<RecordsVo>? = null
    var total: Int? = null
    var size: Int? = null
    var current: Int? = null
    var orders: List<JSONObject>? = null
    var searchCount: Boolean? = null
    var pages: Int? = null
}
